build/helper/metadata_merge_dicts.py

```diff
@@ -6,6 +6,31 @@
 pp = pprint.PrettyPrinter(indent=4, width=80)
 
 
+def merge_helper(metadata, metadata_type, config):
+    metadata_module = 'metadata.{0}_addon'.format(metadata_type)
+    if 'modules' in config and metadata_module in config['modules']:
+        for m in dir(config['modules'][metadata_module]):
+            if m.startswith('{0}_'.format(metadata_type)):
+                merge_dicts(metadata, config['modules'][metadata_module].__getattribute__(m))
+            # We need to explicitly copy new entries
+            if m == '{0}_additional_{0}'.format(metadata_type):
+                outof = config['modules'][metadata_module].__getattribute__(m)
+                for a in outof:
+                    metadata[a] = outof[a]
+
+    # Delete any entries that are empty
```

Comment:
When would this happen, can you provide context in the comment?

Reply:
See `DigitsResolution` in nidmm's enum_addon.